sdk/src/types/block/output/mod.rs

```diff
@@ -417,6 +417,17 @@ impl Output {
 
         Ok(())
     }
+
+    /// Return the output kind string of an [`Output`].
+    pub fn kind_str(&self) -> &str {
```

Comment:
I'd put the function after the `kind` one

Reply:
fixed